### ISSUE TITLE
In `Machine Inventories`, the `Import YAML` button is displayed as an icon rather than the words `Import YAML`

### ISSUE THREAD

**Issue by markusewalker** [rancher/elemental-ui#8]
**Setup**
- Rancher version: `v2.6.8`
- Browser type & version: Chrome

**Describe the bug**

In the `Machine Inventories` page, currently here is the view:

![image](https://user-images.githubusercontent.com/52591819/189418750-18560582-382a-448d-8d06-2e8760bff421.png)

In the top-right corner, we have the buttons of `Import YAML`, `Create from YAML` and `Add Filter`. When you hover over the first button, it shows that the button is `Import YAML`. Rather than having an icon detonate this, it would be better if we have the button explicitly state `Import YAML`.

We're already stating what the name of the button is as soon as you hover it, so it makes sense to just have the name shown as `Import YAML` rather than the import icon.

**To Reproduce**

1. Setup Rancher and ensure that `elemental-operator` is applied to access OS Management.
2. Navigate to OS Management > Create Machine Inventories.
     - If needed, create a machine registration first, so that you can access.
3. Note that the `Import YAML` is an icon rather than the phrase `Import YAML` itself.

**Result**
There is an icon that detonates the operation `Import YAML`.

**Expected Result**
The button should simply state `Import YAML` instead of being an icon that signifies what the operation is doing.

**Comment by juadk** [comment]
ok but the button is disabled
![image.png](https://zube.io/files/rancher/fcab29a9183a8cfd5d3559dec685bbb0-image.png)
What event enables it?

**Comment by aalves08** [comment]
Hi @juadk . Been taking a look at that issue with the YAML import button…. It’s disabled because it didn’t met a certain condition and now I know why: restricted users such as standard users don’t have access to all clusters so we had to introduce the concept of a “blank cluster” in order for the extension to be reachable to these users without specific access to a certain cluster.
By doing so, the YAML import would never work because it doesn’t know to which cluster apply those YAML(s). The logic is totally bound to a cluster, which a standard user might never have access to.

We will need to remove this button/action.

I have put up a PR https://github.com/rancher/elemental-ui/pull/35 where I have included this change 🙏
